templates/vue/src/config/routes.js

```diff
@@ -57,6 +58,16 @@ export default [
           },
         ],
       },
+      {
+        path: `user-answers`,
+        redirect: `${ROOT_PATH}/user-answers/answers`,
+        children: [
+          {
+            path: `/:tab`,
+            name: names.USERANSWERS,
+          },
+        ],
+      },
```

Comment:
I would just like to make sure that these additions are necessary, since the User Answers modal seems to open at #/answers and not #/nodes/x/user-answers/answers.

Reply:
Thank you for fixing this! To me, it looks good as-is. We might like to add more tabs in the future. Also, since the UI contains a tab, it seems nice that the route is consistent with that structure (similar to user-settings/theme, even though it only has one tab as well).